src/index.html

```diff
@@ -42,15 +42,29 @@
           </a>
         </div>
         <div class="header__item">
-          <a class="header__lang" href="">EN</a>
+          <a
+            class="header__lang lang-disabled"
```

Comment:
```suggestion
            class="header__lang header__lang--disabled"
```

нам потрібно тільки модифікатор тут